docs/UserGuide.adoc

```diff
@@ -468,6 +470,24 @@ The `undo` command fails as there are no undoable commands executed previously.
 `clear` +
 `undo` (reverses the `clear` command) +
 `u` (reverses the `delete 1` command)
+// end::undoredo[]
+
+// tag::view[]
+=== Viewing a different tab : `view OR v`
+
+Change the tab view to show either profiles, tasks, or exercises. +
+Format: `view TYPE OR v TYPE`
+
+****
+* Change the tab view to specified `TYPE`.
+* The type refers to the tab name.
+* The tab name must be `profile`, `task`, or `exercise`.
```

Comment:
just to raise a question. maybe we can rename the tabs to "1. profile" "2. task" "3. exercise", then they specify either the index or the tab name to switch tab. Will it be a bit easier for the users? (I'm not sure actually)

Reply:
Personally I would find using only tab names to be the more intuitive choice (think navigation of directory and folders in cli). Sometimes having more options might not be the best if not necessary as it may introduce confusion or complexity. Would welcome more discussion on this though!